### PR TITLE
Update Ribose's RPM signing key to the actual one.

### PR DESCRIPTION
**Pull request by ni4**
As title says.

### CODE REVIEW

**Review by antonsviridenko — APPROVED** [review]

**Comment by antonsviridenko** [comment]
Previous key [EXPIRED 2021-06-16].
How did it work before? I mean `linux_prepare_ribose_yum_repo()`

**Comment by ni4** [comment]
> Previous key [EXPIRED 2021-06-16].
> How did it work before? I mean `linux_prepare_ribose_yum_repo()`

While it is expired, RPMs were signed when the key was valid.

File: ci/lib/install_functions.inc.sh [hunk 1]
```diff
@@ -117,7 +117,7 @@ netbsd_install() {
 }
 
 linux_prepare_ribose_yum_repo() {
-  "${SUDO}" rpm --import https://github.com/riboseinc/yum/raw/master/ribose-packages.pub
```
Comment (ribose-jeffreylau) [inline ci/lib/install_functions.inc.sh:120]: I'm tempted to add both public keys (`ribose-packages.pub` and `ribose-packages-next.pub`) so when time comes for another key update, these scripts do not need changes.

File: docs/develop/packaging.adoc [hunk 2]
```diff
@@ -43,7 +43,7 @@ yum -y install git cmake3 make gcc-c++
 yum -y install bzip2-devel zlib-devel json-c12-devel
 
 # botan
-rpm --import https://github.com/riboseinc/yum/raw/master/ribose-packages.pub
```
Comment (ribose-jeffreylau) [inline docs/develop/packaging.adoc:46]: ditto

File: docs/installation.adoc [hunk 3]
```diff
@@ -42,7 +42,7 @@ at GitHub.
 
 [source,console]
 ----
-rpm --import https://github.com/riboseinc/yum/raw/master/ribose-packages.pub
```
Comment (ribose-jeffreylau) [inline docs/installation.adoc:45]: ditto

File: ci/lib/install_functions.inc.sh (hunk 1 above)
Comment (ni4) [reply ci/lib/install_functions.inc.sh:120]: @ribose-jeffreylau Thanks, fixed and re-pushed.

**Review by ribose-jeffreylau — APPROVED** [review]
Cheers!

**Comment by ronaldtse** [comment]
Thank you guys!

**Comment by ronaldtse** [comment]
@ni4 let's also resolve the remaining comments from @ribose-jeffreylau , thanks!

**Comment by ni4** [comment]
@ronaldtse Already resolved, just didn't write back.

**Comment by ronaldtse** [comment]
Thanks @ni4 !

**Pull request merged**